libraries/BLE/examples/notification/notification.ino

```diff
@@ -0,0 +1,81 @@
+
+#include "ArduinoBLE.h"
+#include "BLEAttribute.h"
+#include "BLECharacteristicImp.h"
+#include "BLEProfileManager.h"
+
+// LED pin
+#define LED_PIN   13
+
+// create service
+BLEService          ledService("19b10100e8f2537e4f6cd104768a1214");
+
+BLECharacteristic   switchCharacteristic("19b10101e8f2537e4f6cd104768a1214", BLERead | BLEWrite | BLENotify, 1);
+
+BLEDescriptor       switchDescriptor("2901", "switch");
+
+void setup() {
+    Serial.begin(115200);
+    Serial.println("test---");
+    
+    // set LED pin to output mode
+    pinMode(LED_PIN, OUTPUT);
+
+    // begin initialization
+    BLE.begin();
+    Serial.println(BLE.address());
+    
+    // set advertised local name and service UUID
+    BLE.setLocalName("LED");
+    BLE.setAdvertisedServiceUuid(ledService.uuid());
+    
+    // add service and characteristic
+    BLE.addService(ledService);
+    ledService.addCharacteristic(switchCharacteristic);
+    switchCharacteristic.addDescriptor(switchDescriptor);

```

Comment:
No. It will not works. For the design is the proxy doesn't manage the real relationship. So if reverse, it will only have service. Other information will lost.